keps/sig-windows/20190103-windows-node-support.md

```diff
@@ -280,13 +281,20 @@ At a high level, these OS concepts are different:
 - Identity - Linux uses userID (UID) and groupID (GID) which are represented as integer types. User and group names are not canonical - they are just an alias in /etc/groups or /etc/passwd back to UID+GID. Windows uses a larger binary security identifier (SID) which is stored in the Windows Security Access Manager (SAM) database. This database is not shared between the host and containers, or between containers.
 - File permissions - Windows uses an access control list based on SIDs, rather than a bitmask of permissions and UID+GID
 - File paths - convention on Windows is to use `\` instead of `/`. The Go IO libraries typically accept both and just make it work, but when you're setting a path or commandline that's interpreted inside a container, `\` may be needed.
+- Signals - Windows interactive apps handle termination differently, and can implement one or more of these:
```

Comment:
What does the container runtime (docker, containerd) send in order to gracefully terminate the application?

I found this issue:
https://github.com/moby/moby/issues/25982

Does Kubernetes terminationGracePeriodSeconds work as expected?